lib/pymedphys/_mosaiq/connect.py

```diff
@@ -39,7 +37,12 @@ def __init__(
     ):
         try:
             self._connection = pymssql.connect(
-                hostname, username, password, database=database, port=port
+                hostname,
+                username,
+                password,
+                database=database,
+                port=port,
+                read_only=True,
```

Comment:
@sjswerdloff, heads up, this here is an important change to how MOSAIQ connection objects are handled within PyMedPhys (always forced to be read_only)